config.go

```diff
@@ -18,28 +18,28 @@ import (
 // Config complete configuration needed to operate the Client.
 type Config struct {
 	// Name to be used in state channels.
-	Alias        string
-	Address      *Address // OnChain address also called PerunID.
-	DatabasePath string   // Path to the database file.
-	ETHNodeURL   string   // URL of the ETH node. Example: ws://127.0.0.1:8545
-	IP           string   // Ip to listen on.
-	Port         uint16   // Port to listen on.
+	Alias   string
+	Address *Address // OnChain address also calles PerunID.
```

Comment:
```suggestion
	Address *Address // OnChain address and PerunID.
```
there also was a typo introduced "calle_s_"

Reply:
Must have happened in the rebase merging…